components/Entry/index.js

```diff
@@ -0,0 +1,35 @@
+import { styled } from "styled-components";
+
+function Entry({ JPDefinition, JPReading, ENDefinition }) {
+  return (
+    <StyledWordCard>
+      <StyledJPDefinition>{JPDefinition}</StyledJPDefinition>
+      <StyledDefinition>[{JPReading}]</StyledDefinition>
+      <StyledDefinition>{ENDefinition.join(" ⦁ ")}</StyledDefinition>
+    </StyledWordCard>
+  );
+}
+
+const StyledWordCard = styled.section`
```

Comment:
A `<section>` [should have an heading of some kind](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) - make sure it is the case!